kratos/geometries/geometry_data.h

```diff
@@ -126,10 +127,11 @@ class GeometryData
         Kratos_Line3D3,
         Kratos_Point2D,
         Kratos_Point3D,
-        Kratos_Sphere3D1
+        Kratos_Sphere3D1,
+        Kratos_Brep_Surface,
+        Kratos_Brep_Curve
```

Comment:
Sorry that I am late in the review, here. However the PR was just one day online... 

Brep_Curve is actually not the one that you should use here as in the terminology this would refer to curves which are on a curve description and not on a surface description.

Reply:
I thought of using Brep_Curve_On_Surface and so on. However, I thought we can simplify split the Brep geometries into curves and surfaces. We can also change it to the exact terminology if this is preferred.